src/Gitonomy/Git/Parser/DiffParser.php

```diff
@@ -90,8 +90,8 @@ protected function doParse()
 
             $oldName = $oldName === '/dev/null' ? null : substr($oldName, 2);
             $newName = $newName === '/dev/null' ? null : substr($newName, 2);
-            $oldIndex !== null ?: '';
-            $newIndex !== null ?: '';
+            $oldIndex = $oldIndex !== null ?: '';
```

Comment:
The usage of `?:` turns `$oldIndex` into a boolean when not null. This looks unexpected to me.